dnf-testing.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+DESTRUCTIVE_TAGS = set([
+    "@destructive",
+    "@no_installroot",
+])
+PROGPATH = os.path.abspath(os.path.dirname(sys.argv[0]))
+
+
+class MultilineHelpFormatter(argparse.HelpFormatter):
+    '''
+    Help formatter that allows multiline help texts.
+    '''
+    def _split_lines(self, text, width):
+        if '\n' in text:
+            return text.splitlines()
+        return super(MultilineHelpFormatter, self)._split_lines(text, width)
+
```

Comment:
I suspect sider is not picking up this as Python code to check, is it? I'd guess there should be two linebreaks here, also the "hacked" import of consts.py in that other commit should raise a warning... Do you know how to make it pick this file up?